packages/core/src/components/essentials/title.ts

```diff
@@ -35,17 +43,17 @@ export class Title extends Component {
 
 			// add events for displaying the tooltip with the title
 			const self = this;
-			title.on("mouseenter", function() {
+			title.on("mouseenter", function () {
```

Comment:
why do we need a space between `function` and the brackets?

Reply:
i think my auto formatter did that, but not sure why cause it did it nowhere else ❓